accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableinstaller/impl/NotRegularMembers.java

```diff
@@ -0,0 +1,103 @@
+package biz.netcentric.cq.tools.actool.authorizableinstaller.impl;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import javax.jcr.RepositoryException;
+
+import org.apache.jackrabbit.api.security.user.Authorizable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import biz.netcentric.cq.tools.actool.helper.Constants;
+
+/**
+ * Auxiliar class to store a list of non regular members.
+ */
+class NotRegularMembers {
```

Comment:
what are non-regular members? Please clarify in javadoc.

Reply:
maybe a better name would be to use a positive name, e.g. `GroupsAndSystemUserMembers`